.changesets/fix_geal_response_validation_errors.md

```diff
@@ -0,0 +1,5 @@
+### add errors for response validation ([Issue #5372](https://github.com/apollographql/router/issues/5372))
```

Comment:
This PR has [a bug](https://github.com/apollographql/router/pull/5787/files#r1851078537); the next version should not be released until it is fixed.

Reply:
We have reverted the PR.  It will re-surface at a later date.